docs/usage/faq.md

```diff
@@ -0,0 +1,272 @@
+1300	|	186826	|	family	|		|	0	|	1	|	11	|	1	|	0	|	1	|	0	|	0	|	code compliant	|
+1301	|	1300	|	genus	|		|	0	|	1	|	11	|	1	|	0	|	1	|	0	|	0	|	code compliant	|
+1311	|	1301	|	species	|	SA	|	0	|	1	|	11	|	1	|	0	|	1	|	1	|	0	|	code compliant; specified	|
+```
+
+### accession2taxid
+
+An accession2taxid file is a file that maps database sequence accession IDs (e.g. NCBI GenBank) to their corresponding taxon IDs.
+
+From NCBI, these would correspond to `nucl_gb.accession2taxid.gz` and `prot.accession2taxid.gz` from the [NCBI taxonomy FTP server](https://ftp.ncbi.nlm.nih.gov/pub/taxonomy/accession2taxid/).
+
+It is formatted as a four column tab-separated file **with** a header row.
+
+| Column | Column Name      | Description                                                                                                                                                                                                     |
+| ------ | ---------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 1      | accession        | The accession ID of the sequence without a version - typically embedded as first part of a FASTA header entry (e.g. for `>NC_012920 Homo sapiens mitochondrion, complete genome)` the accession is `NC_012829`) |
+| 2      | accesion.version | The full accession version of the accession with the version suffix (e.g. `.1`)                                                                                                                                 |
+| 3      | taxid            | The taxon ID corresponding to the names/nodes.dmp                                                                                                                                                               |
+| 4      | gi               | The old-style ([now deprecated](https://ncbiinsights.ncbi.nlm.nih.gov/2016/12/06/converting-gi-numbers-to-accession-version/)) NCBI GenInfo Identifier                                                          |
+
+Example:
+
+```tsv title=nucl_db.accession2taxid
+accession	accession.version	taxid	gi
+NC_018507	CP003708.1	91844	401871806
+NZ_CP019811	CP019811.1	1311	1328913993
+NZ_CP069563	CP069563.1	817	1986148684
+NZ_CP069564	CP069564.1	817	1986152744
+NC_012920	J01415.2	9606	113200490
+NZ_LS483480	LS483480.1	727	1403648431
+MT192765	MT192765.1	694009	1821109001
+```
+
+It's important to note that you can have multiple accession numbers within a FASTA file (i.e., one per sequence in the file), therefore accession2taxid mapping files should ensure they reference _all_ accession numbers that are present in the FASTA file.
+
+The accession column should not include the accession version in the first column (i.e. `1`)
+
+### nucl2taxid
+
+An accession2taxid file is a file that maps database sequence accession IDs (e.g. NCBI GenBank) to their corresponding taxon IDs.
```

Comment:
you mention `accession2taxid` but title of section is `nucl2taxid`

Reply:
It's serves the same purpose, but I've made a suggestion to make it clearer `accession2taxid-like file` (see below))